collections/StudentGuide.ts

```diff
@@ -0,0 +1,27 @@
+import type { CollectionConfig } from 'payload'
+
+export const StudentGuide:CollectionConfig = {
+    slug: 'studentGuide',
+    upload: {
+        staticDir: 'media',
+        mimeTypes: ['application/pdf'],
+    },
+    fields: [
+        {
+            name: "language",
+            type: "select",
+            hasMany: false,
+            options: [
+                {
+                    label: "Português",
+                    value: "Português",
+                },
+                {
+                    label: "Inglês",
+                    value: "Inglês",
+                }
+            ],
+            required: true,
+        }
+    ]
+}
```

Comment:
Add Portuguese label, for example in language. See others collections for examples

Reply:
I don't get it.